File: src/avaliacaoPPGI/Periodico.java
package avaliacaoPPGI;

class Periodico extends Veiculo {

	private String ISSN;
	
	private static final long serialVersionUID = 1L;
	
	public Periodico(String sigla, String nome, float fatorDeImpacto, String iSSN) {
		super(sigla, nome, fatorDeImpacto);
		this.ISSN = iSSN;
	}
	
	public String getISSN() {
		return ISSN;
	}
	
	public void setISSN(String iSSN) {
		ISSN = iSSN;
	}

	@Override
	public String toString() {
		return "Periodico [sigla=" + getSigla() + ", nome=" + getNome()
		+ ", fatorDeImpacto=" + getFatorDeImpacto() + ", ISSN=" + ISSN + "]";
	}

}

File: julia/cod/Periodico.jl
using Base
include("Veiculo.jl")

struct Periodico <: Veiculo
    @atributos_Veiculo
    issn::String

    Periodico(sigla::String, nome::String, impacto::Float64, issn::String) = 
        new(sigla, nome, impacto, Tuple{Int64, String}[], issn)
end

issn(p::Periodico) = p.issn